src/components/ErrorModal/ErrorModal.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./ErrorModal.css";
+
+const ErrorModal = ({ error, closeModalButton }) => {
+  return(
+    <div className="error-modal" tabIndex="0">
+      <div className="modal-content" role="dialog" aria-modal="true">
+        <p id="errorMessage">{error}</p>
+        <button
+          onClick={() => closeModalButton()}
```

Comment:
For callback functions that don't require arguments, you can pass the function reference into the event handler. `onClick={closeModalButton}`